### DISCUSSION TITLE
[BUG]: 封面图片加载缓慢，通常会有5-10秒的等待时长

### DISCUSSION THREAD

**Discussion by Baneik** [magicdawn/Bilibili-Gate#85] [General]
### 环境

浏览器: (如 Chrome)
浏览器版本: (如 113.0.5672.126)
脚本管理器: (如 暴力猴 / TamperMonkey)
脚本版本: (如 0.14.0, 请先更新到最新版本!!!)

### 问题详细描述

封面图片加载缓慢，通常会有5-10秒的等待时长
![image](https://github.com/magicdawn/bilibili-app-recommend/assets/111330534/791293df-a7eb-4ee4-b857-f583da4487eb)
没有发现触发规律，刷新就可能会有，频率不高大概两三天会遇到一次，每次持续时间在几十秒到一两分钟，疑似是风控机制？
但是现象的规律有如下：
B站的封面有三个服务器，i0-i2，出现延迟情况都是其中某一个服务器延迟，比如一次刷新加载10张图片，其中三张来自i0，此时发生延迟，这三张都会等待较长的时间才会响应。
但是B站首页原本的推荐没有出现过类似的问题，并且在出现该问题的间隙，我曾尝试过快速的加载B站自带的推荐，也并没有延迟，在测试完B站自带推荐后，继续刷新脚本的推荐，依旧存在延迟。

这个问题比较玄学，我也不好说这到底是不是个BUG，只是提一嘴，有这个现象。

**Comment by magicdawn** [comment]
可能是转 avif 比较卡, CI 版本加了一个开关, 可以关掉 avif 试试

优先 avif, 其次 webp, 其次 jpg
